pi-manage

```diff
@@ -1319,6 +1282,250 @@ def import_tokens(file, tokenrealm=None):
+    """
+    import privacyIDEA configuration from file
+    """
+    if filename:
+        with open(filename, 'r') as f:
+            contents = f.read()
+    else:
+        filename = "Standard input"
+        contents = sys.stdin.read()
+
+    contents_var = ast.literal_eval(contents)
+
+    # be backwards-compatible. In old versions of pi-manage config were exported to
+    # individual files as python list without dict key
+    if isinstance(contents_var, list):
+        conftype_list = [conftype]
+        contents_var = {conftype: contents_var}
+    else:
+        if conftype:
+            conftype_list = [conftype]
+        else:
+            conftype_list = list(contents_var.keys())
+
+    for conftype in conftype_list:
+
+        print("Importing {0!s} from {1!s}".format(conftype, filename))
+
+        config_list = contents_var[conftype]
+
+        if conftype == "policy":
+            cls = PolicyClass()
+        elif conftype == "resolver":
+            pass
+        elif conftype == "event":
+            cls = EventConfiguration()
+
+        if cleanup:
+            print("Cleanup old {0!s}.".format(conftype))
+            if conftype == "policy":
+                policies = cls.list_policies()
+                for policy in policies:
+                    name = policy.get("name")
+                    r = delete_policy(name)
+                    print("Deleted policy {0!s} with result {1!s}".format(name, r))
+            elif conftype == "resolver":
+                # Todo: Implement resolver cleanup on import
+                print("No cleanup for resolvers implemented")
+            elif conftype == "event":
+                events = cls.events
+                for event in events:
+                    name = event.get("name")
+                    r = delete_event(event.get("id"))
+                    print("Deleted event '{0!s}' with result {1!s}".format(name, r),
+                          file=sys.stderr)
+
+        for config in config_list:
+            action_str = "Added"
+            if conftype == "policy":
+                name = config.get("name")
+                exists = cls.list_policies(name=name)
+            elif conftype == "resolver":
+                name = config.get("resolvername")
+                exists = get_resolver_list(filter_resolver_name=name)
+            elif conftype == "event":
+                # Todo: This check does not work properly. The event is created nevertheless
+                name = config.get("name")
+                events_with_name = [e for e in cls.events if (name in e.get("name"))]
+                if events_with_name:
+                    exists = True
+                    event_id = events_with_name[0].get("id")
+                else:
+                    exists = False
+                    event_id = None
+            else:
+                print("Warning: conftype {0!s} not in policy, event and resolver".format(conftype))
+            if exists:
```

Comment:
`exists` and `name` might not be initialized here. Please check.

Reply:
In principle this is not possible, but I initialized them properly.